### PR TITLE
fix keadm upgrade If edgecore is stopped, the keadm process will stop

### PR DESCRIPTION
**Pull request by wlq1212**
**What type of PR is this?**
/kind bug

**What this PR does / why we need it**:
Use killMode to control the policy, repair keadm upgrade If edgecore is stopped, the keadm process will stop
image

**Which issue(s) this PR fixes**:

Fixes https://github.com/kubeedge/kubeedge/issues/4768
Fixes https://github.com/kubeedge/kubeedge/issues/5094

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Review by daixiang0 — APPROVED** [review]
LGTM

**Comment by wlq1212** [comment]
cc cc @fisherxu @WillardHu @ZhengXinwei-F

File: keadm/cmd/keadm/app/cmd/common/content.go [hunk 1]
```diff
@@ -34,6 +34,7 @@ ExecStart=%s
 Restart=always
 RestartSec=10
 Environment=%s
+KillMode=process
```
[the remark this answers is not in the document]
Comment (wlq1212) [reply keadm/cmd/keadm/app/cmd/common/content.go:37]: There will be a corresponding FAQ guide

**Review by ZhengXinwei-F — APPROVED** [review]
lgtm

File: keadm/cmd/keadm/app/cmd/common/content.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (fisherxu) [reply keadm/cmd/keadm/app/cmd/common/content.go:37]: Let's open an issue in website repo to track, then add the docs in node ugrade part, thanks :) @wlq1212

**Review by fisherxu — COMMENTED** [review]
/approve
/assign @WillardHu

**Pull request merged**